station/utils/observations.py

```diff
@@ -0,0 +1,151 @@
+        if clean and status == ObservationStatus.SUCCESS:
+            print(f"obsdir={obsdir} Cleaning useless files from observation {d.name}...")
+            obs_clean(obsdir, d.name)
+        if del_uploaded and status == ObservationStatus.SUBMITTED:
+            print(f"obsdir={obsdir} Deleting uploaded observation {d.name}...")
+            obs_del(obsdir, d.name)
+
+    print(f"SUMMARY: {total_cnt} observations, {submitted_cnt} uploaded, {success_cnt} successful, {useless_cnt} useless, {unknown_cnt} unknown.")
+
+
+def obs_print_info(obsdir: str, obsname: str):
+    """ Prints information about observation in the specified directory."""
+    du = sum(file.stat().st_size for file in Path(obsdir + "/" + obsname).rglob('*'))
+
+    regex = re.compile(r"([0-9]{4}-[0-9]{2}-[0-9]{2}-[0-9]{4})-([a-zA-Z-0-9_]+)")
+    match = regex.match(obsname)
+    date = ""
+    satname = ""
+    if match:
+        date = match.group(1)
+        satname = match.group(2)
+
+    status = obs_determine_status(obsdir, obsname)
+    print(f"Observation {obsname} (date: {date}, sat: {satname}) uses {sizeof_fmt(du)} of disk space, status: {status}.")
+
+
+def obs_determine_status(obsdir: str, obsname: str) -> ObservationStatus:
+
+    files = [f for f in Path(obsdir + "/" + obsname).glob('*') if f.is_file()]
+
+    # Test 1: If there's a upload status file, it's a successful, uploaded observation.
+    for file in files:
+        if file.name == "uploaded.json":
+            return ObservationStatus.SUBMITTED
+
+    # Test 2: check if there is only a session.log file. If there is, there's no data, so it's a failed observation.
+    # If there's only one file and it's just a log, it's a failed observation.
+    if len(files) == 1 and files[0].name == "session.log":
+        return ObservationStatus.USELESS
+
+    # Test 3: Check if there's a signal.wav file that's too small. If there is, it's a failed observation.
+    for file in files:
+        if file.name == "signal.wav" and file.stat().st_size < 1024:
+            return ObservationStatus.USELESS
+
+    # Test 4: Check if there's a large png file. If there is, it's a successful observation.
+    found = False
+    for file in files:
+        if str(file.name).endswith('png'):
+            found = True
+            break
+
+    if found and file is not None:
+        if file.stat().st_size < 1024:
+            # Found a small png file {file}, assuming this is a failed observation.
+            return ObservationStatus.USELESS
+        else:
+            # Found a large png file {file}, assuming this is a successful observation.
+            return ObservationStatus.SUCCESS
+
+    # Test 5: If there's a very small wav file, it's useless junk.
+    if len(files) == 1 and files[0].name == "signal.wav" and files[0].stat().st_size < 1024:
+        return ObservationStatus.USELESS
+
+    return ObservationStatus.UNKOWN
+
+
+def sizeof_fmt(num, suffix="B"):
+    for unit in ("", "K", "M", "G", "Ti", "Pi", "Ei", "Zi"):
```

Comment:
You use the denominator equal to 1000. It would be best if you used the prefixes without "i" as "T", "P", "E", "Z".